apollo-federation/tests/subgraph/subgraph_validation_tests.rs

```diff
@@ -256,8 +256,9 @@ mod fieldset_based_directives {
         }
     }
 
+    // TODO: UNSUPPORTED_ON_INTERFACE
     #[test]
-    #[should_panic(expected = r#"subgraph error was expected:"#)]
+    #[should_panic(expected = r#"Mismatched errors:"#)]
```

Comment:
We are generating a spurious `EXTERNAL_UNUSED` error now.
I have a follow-up PR to fix that, since it's related to directive reference metadata.